docs/root/intro/version_history.rst

```diff
@@ -1,22 +1,24 @@
 Version history
 ---------------
 
-1.7.0
-=====
+1.7.0 (Pending)
+===============
 
 * Added :ref:`weighted round robin
   <arch_overview_load_balancing_types_round_robin>` support. The round robin
   scheduler now respects endpoint weights and also has improved fidelity across
   picks.
-* admin: added :ref:`/config_dump endpoint <operations_admin_interface_config_dump>` for dumping current configs
+* access log: ability to format START_TIME
+* admin: added :http:get:`/config_dump` for dumping current configs
+* admin: added :http:get:`/stats/prometheus` as an alternative endpoint for getting stats in prometheus format.
 * admin: added :ref:`/runtime_modify endpoint <operations_admin_interface_runtime_modify>` to add or change runtime values
+* admin: mutations must be sent as POSTs, rather than GETs. Mutations include:
+  :http:post:`/cpuprofiler`, :http:post:`/healthcheck/fail`, :http:post:`/healthcheck/ok`,
```

Comment:
WDYT of using this style of live cross-reference here, which I found in fs_flags.rst?

Reply:
Sure, as long as it links up OK.